src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -25,7 +25,8 @@ import java.util.*
 
 internal class LincheckClassVisitor(
     private val transformationMode: TransformationMode,
-    classVisitor: ClassVisitor
+    classVisitor: ClassVisitor,
+    private val ideaPluginEnabled: Boolean
```

Comment:
Why do you need to pass it as an argument? This property is system-wide, you can just read it.